assets/src/stories-editor/blocks/amp-story-page/background-color-settings.js

```diff
@@ -23,9 +23,14 @@ import {
  * @param {Array} props.backgroundColors Current background colors.
  * @param {Function} props.setAttributes setAttributes callback.
  * @param {number} props.overlayOpacity Overlay opacity.
+ * @param {number} props.hasMedia Whether the page has a media item set.
  * @return {ReactElement} Component.
  */
-const BackgroundColorSettings = ( { backgroundColors, setAttributes, overlayOpacity } ) => {
+const BackgroundColorSettings = ( { backgroundColors, setAttributes, overlayOpacity, hasMedia } ) => {
+	const hasColors = backgroundColors
```

Comment:
Could be simplified to:

`const HasColors = backgroundColors.some ( ( color ) => Object.values( color ).some( Boolean ) );`

Reply:
Much better!

Reply:
Went with the first one but used `color && Object.values( color ).some( Boolean )` as `color` it's not guaranteed to be an object